src/Pimple/Container.php

```diff
@@ -58,6 +63,22 @@ public function __construct(array $values = array())
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function get($id)
```

Comment:
This conflict with [`Silex\Application::get()`](https://github.com/silexphp/Silex/blob/v2.0.3/src/Silex/Application.php#L143). That's a huge backward compatibility break for both Pimple and Silex.